README.rst

```diff
@@ -31,10 +31,8 @@ Maintained Major Branches
 ====== ==================== ===============
 Major  Initial release date Supported until
 ====== ==================== ===============
-v1.x.x v1.4.0_: 24/06/2016  Fri 08/12/2017
 v2.x.x v2.0.0_: 08/09/2017  TBD
 ====== ==================== ===============
-.. _v1.4.0: https://github.com/sociomantic-tsunami/makd/releases/tag/v1.4.0
```

Comment:
Wouldn't it be a good opportunity to set the end of life of V2 too? Given the circumstances I guess a regular period of months is not really necessary.

Reply:
So, immediately?

Reply:
I mean, it is almost unmaintained anyway. Also, the fact that it is unsupported doesn't really mean that you can backport some fix if it is easy and useful enough, but *guaranteeing* a 6 months maintenance period (or so) seems excessive given the current state of the project, IMHO. I would make only a *best-effort* promise. :shrug:

Reply:
Also, it in unlikely that in the next 3 months a severe bug is found that it is really important to get backported, so in practice for bug fixes it doesn't seem like a big deal. New features or backward compatible changes have a slight higher chance, IMHO, and in this case if v2 is still maintained, new features should go there, which means then need to be merged to v3 and make merge trains, etc. That also looks like an unnecessary overhead.

Also, it probably makes sense to reduce the maintenance guarantee for minor version to maintaining only the latest one.